dll/win32/shell32/dialogs/filedefext.cpp

```diff
@@ -717,6 +715,19 @@ CFileDefExt::GeneralPageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lPar
             }
             break;
         }
+        case PSM_QUERYSIBLINGS:
+        {
+            // reset icon
+            HWND hIconCtrl = GetDlgItem(hwndDlg, 14025);
```

Comment:
Do we have a resource constant for this hardcoded number?

Reply:
No, yet. I will add them.

Reply:
@HBelusca looks like file properties dialog is totally missing any resource constants: https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/shell32/lang/en-US.rc;hb=fafcd50a35a77f1dd1f4606a30dd4f5b6fe520bd#l261

Reply:
As a second option, you can use the macro constants in <dlgs.h>.

Reply:
OK let's do the following: I commit this PR now, and let's create (if you wish) a separate PR where the constants are introduced?

Reply:
Okay, let's go with it.